src/cljs/ataru/virkailija/application/handlers.cljs

```diff
@@ -4,17 +4,20 @@
             [ataru.virkailija.autosave :as autosave]
             [ataru.virkailija.application-sorting :as application-sorting]
             [reagent.core :as r]
-            [taoensso.timbre :refer-macros [spy debug]]))
+            [taoensso.timbre :refer-macros [spy debug]]
+            [ataru.feature-config :as fc]))
 
 (reg-event-fx
   :application/select-application
   (fn [{:keys [db]} [_ application-key]]
     (if (not= application-key (get-in db [:application :selected-key]))
```

Comment:
Apparently returning `nil` from a event handler does nothing. App state is preserved and re-frame says nothing. Let's call it a feature?